### DISCUSSION TITLE
Agricultural land classification

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#67] [Planning consideration]
## Agricultural land classification

Agricultural land classification is a national dataset that is already available.

The [Geospatial commission have included it on their list of important datasets for housing and planning](https://www.gov.uk/government/publications/bringing-together-housing-land-and-planning-geospatial-datasets). And, we have identified it as a potential fit for [planning.data.gov.uk](https://www.planning.data.gov.uk/) because it might help to identify "Best and Most Versatile (BMV) agricultural land" so that it is protected from significant, inappropriate or unsustainable development proposals.

If you have any thoughts on this dataset please let us know.

**Comment by colmjude** [comment]
The available national dataset, Provisional Agricultural Land Classification dataset, is published by Natural England. It was last updated in 2019 and contains 1826 records.

Lots of the records have the same identifier so we assume they are the individual polygons of a multi-polygon. Similar to the national nature reserves (on the [natural england portal](https://naturalengland-defra.opendata.arcgis.com/datasets/Defra::national-nature-reserves-england/about) - 1109 records vs on [planning.data.gov.uk](https://www.planning.data.gov.uk/dataset/national-nature-reserve) - 219 records)

**Reply by colmjude** [reply]
Here are the fields contained in the dataset and example values.

| field | example | map to |
| --- | --- | --- |
| GEOGEXT | TG2 | reference |
| AREA | 3,712.535 | |
| ALC_GRADE | Grade 1 | agricultural-land-classification-grade |
| PERIMETER | 69,726.523 | |
| Shape__Area | 37,125,350 | |
| Shape__Length | 69,726.523 | |

The majority of the fields are derived data so we would ignore those for now.

A potential schema given the available fields could be

| field | ALC field | default value |
| --- | --- | --- |
| reference | GEOGEXT | |
| geometry | | |
| organisation | | government-organisation:PB202 |
| agricultural-land-classification-grade | ALC_GRADE | |
| notes | | | 
| entry-date | | |
| start-date | | |
| end-date | | |

**Reply by colmjude** [reply]
As well as defining an `agricultural-land-classification` dataset we will likely need to define a `agricultural-land-classification-grade` dataset to list the possible grade values.

**Reply by colmjude** [reply]
Schema for [agricultural-land-classification](https://github.com/digital-land/specification/blob/main/content/dataset/agricultural-land-classification.md?plain=1)

**Comment by drewhardy77** [comment]
See discussion on[ Best and Most Versatile Land ](https://github.com/digital-land/data-standards-backlog/discussions/115) for an update to national data. The data now available (Predictive Agricultural Land Classification) is  a new dataset, not an update on the existing Provisional Agricultural Land Classification, though the schema will  be similar
